undent_test.go

```diff
@@ -462,3 +519,27 @@ func BenchmarkString(b *testing.B) {
 		})
 	}
 }
+
+func captureStdout(f func()) string {
```

Comment:
as this is a test helper, could pass in t ? ( i know the os calls should never error but makes a nicer debug if they do ) 
```suggestion
func testHelperCaptureStdout(t testing.T, f func()) string {
	r, w, err := os.Pipe()
	if err != nil {
		t.Log("test helper failed")
		t.FailNow()
	}
}
```

Reply:
since this is a tester you should add `t.Helper()` on it.

Reply:
I did not know about that :)